games-emulation/81_libretro/81_libretro-1.0a.20171221git.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A port of the EightyOne ZX-81 emulator to libretro"
+DESCRIPTION="81-libretro is an work in progress port of the EightyOne \
+(a.k.a. THE Sinclair Emulator) to libretro. It's being developed on Windows \
+with MinGW (64 bits) and Debian 8, and tested on RetroArch 1.2 frontend."
+HOMEPAGE="http://www.chuntey.com/"
+COPYRIGHT="2008-2018 Michael D Wynne, the libretro team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="6ae21e0980bb8190fe1fc3cb6c4f00797cc6bf0e"
+SOURCE_URI="https://github.com/libretro/81-libretro/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="3332f2777f4b6cc4a8d4109ba74063d1cd118cced49df91478e14fddc064fe68"
+SOURCE_FILENAME="81-libretro-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="81-libretro-$srcGitRev"
+ADDITIONAL_FILES="81_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	81_libretro$secondaryArchSuffix = $portVersion
+	lib:81_libretro$secondaryArchSuffix = $portVersion
```

Comment:
I don't really know if we really need this line in general.

```diff
-	lib:81_libretro$secondaryArchSuffix = $portVersion
+	addon:81_libretro$secondaryArchSuffix = $portVersion
```

Reply:
:+1: for the line with the `addon:`.
I'd even recommend such a line for every recipe that puts a shared lib in `$addOnsDir/libretro/`.

Reply:
Sure. Let me change that on every core :)